src/content/protocols/sky.md

```diff
@@ -0,0 +1,387 @@
+| ESM                       | [0x09e05fF6142F2f9de8B6B65855A1d56B6cfE4c58](https://etherscan.io/address/0x09e05fF6142F2f9de8B6B65855A1d56B6cfE4c58) | contract |
+| END                       | [0x0e2e8F1D1326A4B9633D96222Ce399c708B19c28](https://etherscan.io/address/0x0e2e8F1D1326A4B9633D96222Ce399c708B19c28) | contract |
+| Jug                       | [0x19c0976f590D67707E62397C87829d896Dc0f1F1](https://etherscan.io/address/0x19c0976f590D67707E62397C87829d896Dc0f1F1) | contract |
+| Pot                       | [0x197e90f9fad81970ba7976f33cbd77088e5d7cf7](https://etherscan.io/address/0x197e90f9fad81970ba7976f33cbd77088e5d7cf7) | contract |
+| DssAutoLine               | [0xc7bdd1f2b16447dcf3de045c4a039a60ec2f0ba3](https://etherscan.io/address/0xc7bdd1f2b16447dcf3de045c4a039a60ec2f0ba3) | contract |
+
+<!-- Contracts with mint rights on MKR:
+End : 0x0e2e8F1D1326A4B9633D96222Ce399c708B19c28
+ClipperMom: 0x79FBDF16b366DFb14F66cE4Ac2815Ca7296405A0
+DSPause: 0xbE286431454714F511008713973d3B053A2d38f3
+
+VAT: 0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B 0x65fae35e
+Spotter 0x65c79fcb50ca1594b025960e539ed7a9a6d434a3
+
+Rates modules:
+Pot 0x197e90f9fad81970ba7976f33cbd77088e5d7cf7
+Jug 0x19c0976f590D67707E62397C87829d896Dc0f1F1
+
+DssAutoLine 0xc7bdd1f2b16447dcf3de045c4a039a60ec2f0ba3
+LineMom: 0x9c257e5Aaf73d964aEBc2140CA38078988fB0C10
+
+**ESM**: 0x09e05fF6142F2f9de8B6B65855A1d56B6cfE4c58
+**END**: 0x0e2e8F1D1326A4B9633D96222Ce399c708B19c28
+
+OSM: oracle security module
+OsmMom: 0x76416A4d5190d071bfed309861527431304aA14f
+Liquidation Oracle: 0x88f88bb9e66241b73b84f3a6e197fbba487b1e30
+
+to add or investigate: SkyLink
+
+dog owners:
+LockstakeClipper, Clipper, End
+
+Abacus: -->
+
+## Permissions
+
+During our analysis we discovered a paradigm of _Authority_ contract used to handle fine-grained permissions over functions and contracts.
+In most of the cases the authority is `DSChief` and this allows governance proposals to have non-delayed access to certain functions. Nonetheless, this
+could be misused to grant any arbitrary address those same permissions. We note that because of the lack of _events_ in the `DSChief`'s role granting functions,
+we had to scan all governance proposals to ensure this has not happened in the past, and would encourage that those functions are blocked in future versions of the
+governance contract.
+
+&nbsp;
+
+| Contract                              | Function               | Impact                                                                                                                                                                                                                                                                                                                                                                                                                                                 | Owner                                                                                       |
```

Comment:
similar to the permissions table. currently readers dont have an overview of "externally owned permissions". here we even list permissions which are owned by a contract and where this owner cannot be changed. if we keep everything in here bc its autogenerated by the scanner then I think we should move these tables into an "Appendix" at the end of the review